hyperspy/_components/gaussian2d.py

```diff
@@ -90,3 +109,12 @@ def fwhm_y(self):
     @fwhm_y.setter
     def fwhm_y(self, value):
         self.sigma_y.value = value / sigma2fwhm
+
+    # Rotation compared to "x"-axis
+    @property
+    def rotation_degrees(self):
+        """ Finds the rotation compared to the "x-axis" """
```

Comment:
Why using degrees instead of mrad?

Maybe a better docstring would be: "Angle between major axis and x-axis"?